packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -133,6 +139,10 @@ export class Context implements FlexTreeContext, IDisposable {
 	 */
 	// TODO: Use this to automatically provide node keys where required.
 	public fieldSource?(key: FieldKey, schema: TreeFieldStoredSchema): undefined | FieldGenerator;
+
+	public get anchorSet(): AnchorSet {
+		return this.forest.anchors;
+	}
```

Comment:
Unnecessary now that we use the forest? Cursory search didn't show any uses but it was _very_ cursory so take with a grain of salt.

Reply:
Yea, definitely unnecessary since the interface no longer requires it, thanks.